java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -321,6 +321,23 @@ public RosterEntry getEntryForId(String id) {
         return null;
     }
 
+     /**
+     * Return a list of RosterEntry which have a 
+     * particular DCC address.
+     *
+     * @param a 
+     * @return an ArrayList of matching entries
+     */
+    public List<RosterEntry> getEntriesByDccAddress(String a) {
+        List<RosterEntry> list = new ArrayList<>(); 
```

Comment:
See SteveRawlinson/JMRI#1 for the suggested refactoring and test case.  @SteveRawlinson, if that addition is OK with you, could you merge that PR, which will include it here automatically.